### DISCUSSION TITLE
Companies / Apps using Nhost

### DISCUSSION THREAD

**Discussion by elitan** [nhost/nhost#1133] [Show and tell]
If you're using Nhost as the backend for your app, you can share the URL here!

I'll start:

Nhost - The Open Source Firebase Alternative with GraphQL
http://nhost.io/

(Yes we're building Nhost on Nhost)

**Comment by kylebuildsstuff** [comment]
Checkout Buddy – Boost sales with checkout cross sells and upsells (for Shopify Plus merchants)

https://apps.shopify.com/checkout-buddy

Copybuddy – AI copywriting tool for product descriptions, blogs, marketing

https://apps.shopify.com/copybuddy

**Comment by chrtze** [comment]
Hi! We are a small team from Berlin working on a [React library for building node-based UIs](https://github.com/wbkd/react-flow). We are using nhost for our [supporter platform](https://pro.reactflow.dev/) and are super happy with it.

Our platform was built by a single frontend developer - a task that would have never been possible without nhost. Under the hood we use nextjs, stripe and chakra.

Our website: https://reactflow.dev/

**Comment by mmlngl** [comment]
HTCH ("hatch") – Architectural collaboration platform.

Stack: nhost, [urql](https://formidable.com/open-source/urql/), [jotai](https://jotai.org/), [next.js](https://nextjs.org/). All managed in an [nx repo](https://nx.dev/)

"Pre-Launch" at time of posting.

Our site: https://www.htch.app/

**Comment by fungilation** [comment]
Midnight Society, web3 AAA FPS game leveraging NFT for founder access and soon, user owned game items.

Infra: Nhost, Vercel.

https://midnightsociety.com/

**Comment by atapas** [comment]
`ReactPlay` is an Opensource platform for React developers to learn, create and share ReactJS Projects. It helps you learn ReactJS faster with a hands-on practice model. It is a collection of projects that you can use to learn ReactJS.

Tech-Stack: React, Nhost, Netlify

Website: https://reactplay.io
Twitter: https://twitter.com/ReactPlayIO
Join Us: https://discord.gg/UfYj4MvW9A

**Comment by altschuler** [comment]
[Reshared.org](https://reshared.org) is an open-source platform for sharing things in local communities (like stuff you'd borrow from your neighbor). 

Using Nhost, Vercel, Next.js, [repo here](https://github.com/altschuler/reshared)

**Comment by ded-ditat** [comment]
RevTron - Helping software companies grow recurring revenues systematically with machine learning.

https://www.revtron.ai/

**Comment by Godefroy** [comment]
Rolebase is a SaaS for shared governance organized in circles and roles, with meetings, discussions and tasks.

https://www.rolebase.io/

**Reply by mmlngl** [reply]
👀

I was curious to learn more about what you do.

But I don't understand French. :(

**Reply by Godefroy** [reply]
Hey @mmmoli 
I'm talking about it here: https://github.com/nhost/nhost/discussions/1121

The website is only in french for now, I suppose you can translate it with Google Translate.
All of the webapp and its source code are in english.

**Reply by quentin-decre** [reply]
Hi @Godefroy,
So fun to see you using Nhost here ! Tech world is small :-)

**Comment by quentin-decre** [comment]
Jarvi is a CRM for recruiters, fully connected to Linkedin to help them automate most of the boring tasks, and create more values.

https://www.jarvi.tech/

**Comment by TaylorFacen** [comment]
I use nhost for [Finta](https://finta.io)

Finta lets you sync your account balances, transactions, and investments into Notion, Airtable, Coda, or Google Sheets for each budgeting in the tools you already use.

**Comment by dionjwa** [comment]
Metapages: connected websites for sharable, editable simulations, workflow, apps, and dashboards

https://app.metapage.io

**Comment by dbj** [comment]
We are working on an application for a customer in the international data privacy and security field.

https://xeticode.com
